code/game/mecha/mecha.dm

```diff
@@ -127,15 +133,16 @@
 	icon_state += "-open"
 	add_radio()
 	add_cabin()
-	add_airtank()
+	if (enclosed)
+		add_airtank()
 	spark_system.set_up(2, 0, src)
 	spark_system.attach(src)
 	smoke_system.set_up(3, src)
 	smoke_system.attach(src)
 	add_cell()
 	START_PROCESSING(SSobj, src)
 	GLOB.poi_list |= src
-	log_message("[src.name] created.")
+	log_message("[src.name] created.", LOG_MECHA)

```

Comment:
theres still more instances of LOG_MECHA

Reply:
oh shit i thought it's only in one file i'll get onto that